constants.js

```diff
@@ -26,6 +26,8 @@ const CHATROOM_URL = `https://api.gitter.im${CHATROOM_SUFFIX_URL}`;
 const SERVER_PREFIX_URL = 'http://127.0.0.1:5000';
 const SERVER_DEPLOY_URL = `${SERVER_PREFIX_URL}/deploy`;
 const SERVER_HOWDOI_PREFIX_URL = `${SERVER_PREFIX_URL}/howdoi?query=`;
+const SERVER_MESSAGE_PARSER_PREFIX_URL = `${SERVER_PREFIX_URL}/message_parser?query=`;
+const JOKES_API_URL = 'http://api.icndb.com/jokes/random';
```

Comment:
Yep. It's no longer needed since it has now been substituted by `JOKES_API_INFO`!